pkg/cli/verifier/testdata/httpbin1_permissive.json

```diff
@@ -1837,12 +1837,12 @@
      }
     },
     {
-     "name": "root-cert-for-mtls-inbound:httpbin/httpbin",
```

Comment:
Is there a reason this was service specific originally? /cc @shashankram

Reply:
My guess is we deemed it more flexible at the time, but with the new builder approach it's pretty trivial to move between the 2, and having a single item is both easier to conceptualize, less code, and less compute resources.

I'll wait to resolve for @shashankram's comment

Reply:
Originally, each inbound secret would also specify the allowed client SANs within the validation context. Later, the SAN matching was delegated to the network RBAC filter in the inbound listener instead of doing it via the secret config. So the optimization makes sense now - it was something that was missed during the refactoring of this code before.